### PR TITLE
YJDH-699 | test(kesaseteli): add longer timeouts to tests that seem to need them

### PR DESCRIPTION
**Pull request by karisal-anders**
## Description :sparkles:

TL;DR:
 - **Increasing timeouts** in favor of retries (Retries were added in PR #2940 by @sirtawast and now removed in this PR)

### test(kesaseteli): add longer timeouts to tests that seem to need them

remove jest.retryTimes(5) as they should not be needed anymore
with the increased timeouts, if they still fail then it's good to get
the information about them failing rather than thinking that they're
not failing

refs YJDH-690, YJDH-699

## Issues :bug:

[YJDH-690](https://helsinkisolutionoffice.atlassian.net/browse/YJDH-690) "Some kesäseteli tests fail randomly on employer and youth"
[YJDH-699](https://helsinkisolutionoffice.atlassian.net/browse/YJDH-699) "Increase timeouts for the kesäseteli tests that fail randomly on employer and youth"

## Testing :alembic:

Locally tested using multiple (up to 9) simultaneous runs of tests in a loop,
i.e. I opened up to 9 terminals and ran
- `cd frontend/kesaseteli/youth`
- `while yarn test -t "when submitting form" additional_info.test.tsx; do :; done`
or
- `while yarn test additional_info.test.tsx; do :; done`
or a combination of those for the youth UI tests mentioned in [YJDH-690](https://helsinkisolutionoffice.atlassian.net/browse/YJDH-690).

Similarly for employer UI tests mentioned in [YJDH-690](https://helsinkisolutionoffice.atlassian.net/browse/YJDH-690):
- `cd frontend/kesaseteli/employer`
- `while yarn test; do :; done`

So up to 9 terminals running in a loop, only the youth tests in question at a time or combination of them and their subsets, or the employer tests in question.

They didn't fail seem to be failing anymore after the tweakings in this PR.

## Additional notes :spiral_notepad:

[YJDH-690]: https://helsinkisolutionoffice.atlassian.net/browse/YJDH-690?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[YJDH-699]: https://helsinkisolutionoffice.atlassian.net/browse/YJDH-699?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://tet-pr2981.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**ADMIN branch is deployed to platta: https://tet-admin-ui-pr2981.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**APPLICANT branch is deployed to platta: https://helsinkilisa-ui-pr2981.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr2981.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr2981.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-admin-ui-pr2981.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-handler-pr2981.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://tet-youth-ui-pr2981.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tet-youth-ui-pr2981.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://kesaseteli-pr2981.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**EMPLOYER branch is deployed to platta: https://kesaseteli-employer-pr2981.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-employer-pr2981.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://kesaseteli-handler-pr2981.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-handler-pr2981.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**YOUTH branch is deployed to platta: https://kesaseteli-youth-pr2981.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kesaseteli-youth-pr2981.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by karisal-anders** [comment]
For code reviewing I suggest using "Hide whitespace":
image

As it'll make the diff much smaller.

**Review by melniiv — APPROVED** [review]
